data/avro-schema/src/main/resources/avro/net/corda/data/persistence/DeleteById.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "DeleteById",
+  "doc": "Delete an entity by its id.  See also {@link EntityRequest#request}",
+  "namespace": "net.corda.data.persistence",
+  "fields": [
+    {
+      "name": "entityClassName",
+      "type": "string",
+      "doc": "the fully qualified entity class name"
+    },
+    {
+      "name": "id",
```

Comment:
We use `primaryKey` as a name for the `FindEntity` schema. Maybe it's worth bringing them in line?